ios/KMMIC/Sources/Presentation/Views/ProgressHUD.swift

```diff
@@ -8,8 +8,23 @@
 
 import SwiftUI
 
+private struct ProgressHUDModifier: ViewModifier {
+
+    let isPresented: Binding<Bool>
+
+    func body(content: Content) -> some View {
+        ZStack {
+            content
+            ProgressHUD(isPresented: isPresented)
+        }
+        .ignoresSafeArea()
+    }
```

Comment:
Optional: I prefer to use `overlay` instead
```suggestion
    func body(content: Content) -> some View {
    	return content
			.overlay {
				ProgressHUD(isPresented: isPresented)
			}
    }
```

Reply:
@suho This `overlay` is only available from 15.0+ 🙏